software/nosql/src/main/java/org/apache/brooklyn/entity/nosql/riak/RiakNodeImpl.java

```diff
@@ -99,15 +108,51 @@ public boolean isPackageDownloadUrlProvided() {
 
     @Override
     protected Collection<Integer> getRequiredOpenPorts() {
-        // TODO this creates a huge list of inbound ports; much better to define on a security group using range syntax!
-        int erlangRangeStart = getConfig(ERLANG_PORT_RANGE_START).iterator().next();
-        int erlangRangeEnd = getConfig(ERLANG_PORT_RANGE_END).iterator().next();
+        Integer erlangRangeStart = config().get(ERLANG_PORT_RANGE_START);
+        Integer erlangRangeEnd = config().get(ERLANG_PORT_RANGE_END);
+        sensors().set(ERLANG_PORT_RANGE_START, erlangRangeStart);
```

Comment:
I suspect we get this being set for free, even if you don't include this line - but no strong feelings about leaving it in.